assets/src/dashboard/utils/useFocusOut.js

```diff
@@ -30,7 +30,11 @@ function useFocusOut(ref, callback, deps) {
 
     const onFocusOut = (evt) => {
       // If focus moves somewhere outside the node, callback time!
-      if (evt.relatedTarget && !node.contains(evt.relatedTarget)) {
+      if (
+        evt.relatedTarget &&
+        node.contains(evt.relatedTarget) && // check if node used to have focus
+        !node.contains(evt.currentTarget) // check if focusing away
+      ) {
```

Comment:
This was the magic!

`node` is the container that we want to add the 'focus out' event to (in this case, the context menu)
`evt.relatedTarget` is the element that _previously_ had focus.
 
Before we were checking to see if `!node.contains(evt.relatedTarget)`. So we were checking if **the element that used to have focus** was not in the node.

What we want to check is:
- element that used to have focus was in the node (may not be necessary)
- element that we're focusing is not in the node